internal/testdata/foo/v1/test.proto

```diff
@@ -24,6 +24,7 @@ message RedactedRepeatedField {
 
 message RedactedMap {
   map<string, RedactedMessage> name = 1;
+  map<string, string> not_redacted = 2;
```

Comment:
I added some fields in here to test the cases of "map that does _not_ have message values" and "list that does _not_ contain messages", just to make the tests a little more thorough.

This was also an easy repro case: after adding this field and populating it in the test code (but before applying any fix), the test would panic.